### PR TITLE
Clean `NEWS.md`

### LINKED ISSUE #717: [UAT] re-read and reorganise NEWS entries
Please re-read the NEWS.md file and make it human readable, i.e. re-organise, change order, combine entries, group them if applicable, full sentence, no break lines, no new entries on the old version

### PR DESCRIPTION
**Pull request by Melkiades**
@edelarua do you think we should group better the NAMESPACE polishing and the checkmate stuff? (the second one I think it is much clearer atm). 

Fixes #717

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,61 +1,43 @@
 # tern 0.7.9.9013
 
+### New Features
+* Added stratified Newcombe and Wilson statistics to `estimate_proportion` and
```
Comment (Melkiades) [inline NEWS.md:4]: *stratified Wilson

**Comment by edelarua** [comment]
> @edelarua do you think we should group better the NAMESPACE polishing and the checkmate stuff? (the second one I think it is much clearer atm).
> 
> Fixes #717

Hi @Melkiades, I think the `tern` NEWS is about as simple as we can get it without removing information right now, but feel free to give it a shot if you think you can clean it up a bit!

If you have time to go through and fix grammar checks, I'll be opening new PRs to do the same in `tern.mmrm` and TMC as well!

**Comment by Melkiades** [comment]
> Hi @Melkiades, I think the `tern` NEWS is about as simple as we can get it without removing information right now, but feel free to give it a shot if you think you can clean it up a bit!

If it is fine with you, I think it is ok like it is. In the end, they were major PRs and deserve a chapter in this release imo.
> 
> If you have time to go through and fix grammar checks, I'll be opening new PRs to do the same in `tern.mmrm` and TMC as well!
Done!!

**Review by edelarua — APPROVED** [review]
LGTM! :)

**Pull request merged**